### PR TITLE
Updates from script

### PR DESCRIPTION
**Pull request by mchowning**
Updates from running the script. There are two "extra" comments for the unassociated because one individual claimed the wrong email address, and I [responded to them](https://github.com/WordPress/gutenberg/issues/31913#issuecomment-1072732554).

Please go ahead and merge if you approve.

### CODE REVIEW

File: data/dual-license-responses.json [hunk 1]
```diff
@@ -46,7 +46,11 @@
       954034475,
       966646470,
       966648900,
-      996040374
+      996040374,
+      1064509989,
+      1064524289,
+      1064655585,
+      1072732554
```
Comment (hypest) [inline data/dual-license-responses.json:53]: > 1072732554

That seems to be the comment id of your own question @mchowning , not sure, do we include it to the list as well? 🤔

Comment (mchowning) [reply data/dual-license-responses.json:53]: Every comment that has been "processed" gets added here. This prevents the script from re-prompting about how to handle a given comment every time it is run, and instead the script will only look at "new" comments (i.e., comments not on this list) the next time it has been run. In other words, it is basically the source of truth for what comments we have already looked at and recorded any relevant consents (or denials of consent).

Comment (hypest) [reply data/dual-license-responses.json:53]: Gotcha, thanks for elaborating Matt!

**Review by hypest — APPROVED** [review]
LGTM!

**Comment by hypest** [comment]
> hypest assigned mchowning [34 seconds ago]

Some context on that assignment @mchowning , I see that [the merge commit is attributed to me](https://github.com/WordPress/gutenberg-license/commit/f43eb6165f52d0e25d24e36d0bedb09520b35698) if I go ahead and merge it, but I don't think that attribution is appropriate here since you are doing most of the work. So, I assigned you to "message" you to do the merge from your side. Thanks! 😄

**Comment by mchowning** [comment]
> Some context on that assignment @mchowning , I see that [the merge commit is attributed to me](https://github.com/WordPress/gutenberg-license/commit/f43eb6165f52d0e25d24e36d0bedb09520b35698) if I go ahead and merge it, but I don't think that attribution is appropriate here since you are doing most of the work. So, I assigned you to "message" you to do the merge from your side. Thanks! 😄

👍 No worries at all from my end about getting attribution (there's very little work since it is really just re-running the script most times), but I'm happy to take care of the merge too, so either way is fine.

**Pull request merged**